schemas/README.md

```diff
@@ -35,6 +35,7 @@ Supported keys to describe fields
 - multi\_fields (optional):
 - reusable (optional):
 - index (optional): If `False`, means field is not indexed (overrides type)
+- object_type
```

Comment:
Do you want also to add a line on what is used for? In summary it defines a dynamic template for all the fields inside this object with the above field type.

Reply:
Thanks for this, @codebrain!

Yeah I wrote up this file as I was playing around, to help my faulty memory. It's by no means exhaustive. So good catch :-)